ccmbioinfo/lfs
# CARDS-1785: Set up the HERACLES project

This PR makes the `cards4heracles` project available when starting CARDS. When this project is enabled, the Questionnaires from CARDS-1786 are made available and the UI is restyled for the Heracles project.

To test:

- Build this branch with `mvn clean install`.
- Start the project with `./start_cards.sh --dev --project cards4heracles`
- Visit `http://localhost:8080`. The UI should be styled for Heracles and the appropriate questionnaires should be available.

Also verify this this works in a Docker Compose environment:

- `cd compose-cluster`
- `./cleanup.sh`
- `python3 generate_compose_yaml.py --dev_docker_image --oak_filesystem --cards_project cards4heracles && docker-compose build && docker-compose up -d`
- Visit `http://localhost:8080`. The UI should be styled for Heracles and the appropriate questionnaires should be available.
- `docker-compose down && docker-compose rm && docker volume prune -f && ./cleanup.sh`
